src/pudl/package_data/eia923/column_maps/fgd_operation_maintenance.csv

```diff
@@ -0,0 +1,19 @@
+fgd_labor_supervision_costs_thousand_dollars,,labor_and_supervision,labor_and_supervision,labor_and_supervision,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars,fgd_labor_and_supervision_costs_thousand_dollars
+fgd_land_acquisition_costs_thousand_dollars,land_acquisition,,,,,,,,,,,,,,
+fgd_waste_disposal_costs_thousand_dollars,waste_disposal,waste_disposal,waste_disposal,waste_disposal,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars,fgd_waste_disposal_costs_thousand_dollars
+fgd_maintenance_costs_thousand_dollars,,,,,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars,fgd_maintenance_material_and_other_costs_thousand_dollars
+fgd_maintenance_material_other_costs_thousand_dollars,maintenance_material_other,maintenance_material_other,maintenance_material_other,maintenance_material_other,,,,,,,,,,,
+total_thousand_dollars,total_o_m,total_o_m,total_o_m,total_o_and_m,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars,total_thousand_dollars
+fgd_control_flag,no_fgd_control,no_fgd_control,no_fgd_control,no_fgd_control,,,,,,,,,,,
+fgd_operational_status,fgd_unit_status,fgd_unit_status,fgd_unit_status,fgd_unit_status,,,,,,,,,,,
+fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,fgd_hours_in_service,,,,,,,,,,,
+fgd_sorbent_consumption_1000_tons,quantity_of_sorbent,quantity_of_sorbent,quantity_of_sorbent,quantity_of_sorbent,,,,,,,,,,,
```

Comment:
Good catch. `100pct` and `1000_lbs` are both existing column names in transformed tables, and `1000_tons` comes from the `emissions_control` table. So I will actually standardize to match these, rather than writing everything out to maintain consistency.